samples/base/Logger.hpp

```diff
@@ -86,6 +89,12 @@ namespace darwin {
             /// \param type The log level to set.
             void setLevel(log_type type);
 
+            /// Set the log Level with a given string.
+            ///
+            /// \param level The string that describe the log level wanted. 
+            /// \return true if success, false otherwise.
+            bool setLogLevel(std::string level);
```

Comment:
```suggestion
            bool setLevel(std::string level);
```
Why not use polymorphism ?
Adding multiple name for the same kind of functionality may be confusing.

Reply:
Done